gatsby-browser.js

```diff
@@ -64,6 +64,21 @@ export const onInitialClientRender = () => {
     document.body.appendChild(gtag);
   }
 
+
+
+  if (searchgov && searchgov.affiliate && searchgov.suggestions) {
+    const config = document.createElement('script');
+    config.text = `
+      var usasearch_config = { siteHandle: "${searchgov.affiliate}" };
+    `;
+    document.body.appendChild(config);
+    
+    const src="https://search.usa.gov/javascripts/remote.loader.js";
+    const onLoad = () => console.log("Typeahead suggestions loaded.");
```

Comment:
We don't really need anything to happen onLoad, but I put this in as a placeholder. Feel free to remove.